contributor/05-benefits-of-contribution.md

```diff
@@ -0,0 +1,113 @@
+# Draft comments
+
+Outline for this segment
+
+What are the benefits of contributing - both from an individual as well as from
+a business perspective. 
+
+# Article
+
+## Benefits of becoming an InnerSource Contributor
+
+Contributors are the life blood of Inner Source projects: Every project that is
+run as an Inner Source project comes both with the promise and with the ultimate
+goal of expanding their development team beyond the original founders, tapping
+into the potential for further collaborators amongs users (also sometimes
+referred to as customers in corporations) of that project.
+
+However what would motivate an individual developer to spend time on a project
+that is not under the direction of their manager? What would motivate a manager
+to make time for their developers to improve projects that are not 100% under
+their purview?
+
+### Individual motivation
```

Comment:
One point that is in the [outline](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/outline.md) that I don't seen here is _Varied, fun, and personally fulfilling_.  Is there a way to include it?

Reply:
Thanks.!